install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,426 @@
+            username: "{{ rhsub_user }}"
+            password: "{{ rhsub_pass }}"
+            auto-attach: true
+            disable-repo: ['*']
+            # 'rhel-7-server-rhv-4.2-manager-rpms' supports RHV 4.2 and 4.3
+            enable-repo: ['rhel-7-server-rpms', 'rhel-7-server-extras-rpms', 'rhel-7-server-ansible-2.7-rpms', 'rhel-7-server-ose-3.11-rpms', 'rhel-7-server-supplementary-rpms', 'rhel-7-server-rhv-4.2-manager-rpms']
+          packages:
+            - ansible
+            - ovirt-ansible-roles
+            - openshift-ansible
+            - python-ovirt-engine-sdk4
+  pre_tasks:
+    - name: Create an ssh key-pair for the OCP admin
+      user:
+        name: root
+        generate_ssh_key: yes
+        ssh_key_file: .ssh/id_rsa_ssh_ocp_admin
+
+  roles:
+    - oVirt.image-template
+    - oVirt.vm-infra
+
+- name: post installation tasks on the bastion machine
+  hosts: rhel-bastion
+  tasks:
+    - name: create ovirt-engine PKI dir
+      file:
+        state: directory
+        dest: /etc/pki/ovirt-engine/
+    - name: Copy the engine ca cert to the bastion machine
+      copy:
+        src: "{{ engine_cafile }}"
+        dest: "{{ engine_cafile }}"
+    - name: Copy the secured vars to the bastion machine
+      copy:
+        src: secure_vars.yaml
+        dest: secure_vars.yaml
+        decrypt: false
+    - file:
+        state: directory
+        path: /root/.ssh
+    - name: copy the ocp_admin keypair to the bastion machine
+      copy:
+        src: "{{ item }}"
+        dest: "{{ item }}"
+        mode: 0600
+      with_items:
+        - /root/.ssh/id_rsa_ssh_ocp_admin
+        - /root/.ssh/id_rsa_ssh_ocp_admin.pub
+----
+<1> `<qcow_url>` is the download URL of the *_RHEL-guest-image_*. The *_RHEL-guest-image_* includes the `cloud-init` package, which is required by this playbook. (If you are using a different operating system, you must install the `cloud-init` package separately.)
++
+To copy the `<qcow_url>`:
++
+.. Navigate to link:https://access.redhat.com/downloads/content/69/ver=/rhel---7/latest/x86_64/product-software[].
+.. In the *Product Software* tab, locate the *_Red Hat Enterprise Linux KVM Guest Image_*.
+.. Right-click the *Download Now* button and copy the URL.
++
+This link expires after some time. Save the link when you are ready to run the playbook.
+
+. Create the bastion virtual machine:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# ansible-playbook -i localhost create-bastion-machine-playbook.yaml -e @secure_vars.yaml --ask-vault-pass
+----
+
+. Log in to the Administration Portal and click menu:Compute[Virtual Machines] to verify that the *_rhel-bastion_* virtual machine was created successfully.
+
+== Installing {product-title} on Red Hat Virtualization
+
+You can install {product-title} using the bastion virtual machine:
+
+. Log in to *_rhel-bastion_*.
+. Create an SSH key pair:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# ssh-keygen -f /root/.ssh/id_rsa_ssh_ocp_admin
```

Comment:
@apinnick This needs to happen **before** we run `create-bastion-machine-playbook.yaml` playbook

Reply:
Oh, right-- I see that the key is called by the playbook. Fixed.